value.go

```diff
@@ -197,10 +199,30 @@ func ValueOf(v interface{}) Value {
 		panic("cannot create parquet value from go value of type " + t.String())
 	}
 
-	return makeValue(k, reflect.ValueOf(v))
+	return makeValue(k, nil, reflect.ValueOf(v))
```

Comment:
Should `ValueOf` be modified to handle the case where it's called with a `time.Time`?

Reply:
👍 Added in f13bb96